tests/peer.js

```diff
@@ -371,6 +371,42 @@ describe('Peer', () => {
 
         addConnectionSpy.restore();
       });
+
+      it('should create a new DataConnection, default reliable mode', () => {
+        const addConnectionSpy = sinon.spy(peer, '_addConnection');
+
+        const conn = peer.connect(peerId, {});
+
+        // XXX: property reliable is only visible on Chrome / Firefox
```

Comment:
What do you mean by `XXX`?

Reply:
I often use this means as `it works but I want to find better way, but ...`.

https://en.wikipedia.org/wiki/Comment_%28computer_programming%29#Tags

`dc.reliable` is not visible on Safari and not described in spec.

Reply:
> XXX – warn other programmers of problematic or misguiding code

Understood! thanks :)